.github/workflows/release.yml

```diff
@@ -108,8 +108,7 @@ jobs:
         #   branch-name: master
         #   author-name: evcc-io
         run: |
-          export RELEASE_VERSION=$(echo ${{ env.GITHUB_REF }} | sed -e s#refs/tags/##)
-          sed -e s#0.0.0#$RELEASE_VERSION# packaging/hassio/config.json
+          sed -i -e s#0.0.0#$(echo ${{ env.GITHUB_REF }} | sed -e s#refs/tags/##)# packaging/hassio/config.json
```

Comment:
Das geht nicht. config.json wird im nächsten step mit `"version": "0.0.0"` kopiert. (getested)

Reply:
Yep, hab ich auch gesehen- das dürfte aber daran liegen, dass Du den branch angegeben hast. Geht es Du den raus nimmst (so wie ich das hier nachgereicht habe)? Dann sollte die lokale Datei kopiert werden?

Reply:
bei mir wird der file nur richtig ausgeben bei:
```
        run: |
          export RELEASE_VERSION=$(echo ${{ github.ref }} | sed -e s#refs/tags/v##)
          echo "$RELEASE_VERSION"
          sed -e s#0.0.0#$RELEASE_VERSION# packaging/hassio/config.json
```
so geht es nicht:
```

        run: |
          sed -i -e s#0.0.0#$(echo ${{ env.GITHUB_REF }} | sed -e s#refs/tags/##)# packaging/hassio/config.json
```
CopyCat: Das geht nur mit branch bei mir, da meins "main" ist. Hier sollte es auch ohne branch gehen, da default "master" ist. Oder wo meinst du den "branch" ?

Reply:
wenn Du es mit sed machen willst wisrst du nicht 0.0.0 ersetzen können, sondern die zeile mit "version": "x.x.x" und dann mit "version": "release" ersetzen müssen. Aber den file musst du committen, oder aber direkt in das hassio-addon commiten. Am einfachsten ist es wenn hassio-addon im evcc branche wäre.

Reply:
> wenn Du es mit sed machen willst wisrst du nicht 0.0.0 ersetzen können, sondern die zeile mit "version": "x.x.x" und dann mit "version": "release" ersetzen müssen.

Verstehe ich (leider) nicht. Ich hätte gerne Folgendes:
- Datei hier aktualisieren
- aktualisierte Datei ins hassio Repo committen
- *kein* Commit hier (wenn wir hier committed triggert das den nächsten Build der unnötig ist)

Tut nicht?

Reply:
Wenn wir die Logik erstmal fürs Nightly bauen könnten liesse sich das auch gefahrlos testen. hier müssen wir erst aufs ncähste Release warten...

Reply:
Ich teste das im hintergrund, mit meine repos. Mit sed geht es leider nicht (noch nicht).

Reply:
so der nightly vorschlag ist da: https://github.com/evcc-io/evcc/pull/4394

Reply:
Info:

The simplest option is to clone the target repo, copy the files into the target repo, use the git commandline to stage the files and then commit them. Add the code below in a script step
```

run: |
    git clone https://.:${{ secrets.GITHUB_TOKEN }}@github.com/project target
    rm -rf everything but the .git directory
    copy source\files target
    cd target
    git add . #(add -A)
    git diff-index --quiet HEAD || git commit -m "Automatic publish from github.com/project"
    git push target master
```

Reply:
https://github.com/cpina/github-action-push-to-another-repository/blob/main/entrypoint.sh